apps/src/templates/InputPrompt.story.jsx

```diff
@@ -1,19 +1,19 @@
 import React from 'react';
 import InputPrompt from './InputPrompt';
 
-export default storybook => {
-  return storybook.storiesOf('InputPrompt', module).addStoryTable([
-    {
-      name: 'Input Prompt',
-      description: `Example simple input prompt to take in user  input`,
-      story: () => (
-        <InputPrompt
-          question="What is your quest?"
-          onInputReceived={() => {
-            console.log('user input received');
-          }}
-        />
-      )
-    }
-  ]);
+export default {
+  title: 'InputPrompt',
+  component: InputPrompt
 };
+
+const Template = args => (
+  <InputPrompt
+    question="What is your quest?"
+    onInputReceived={() => {
+      console.log('user input received');
+    }}
+    {...args}
+  />
+);
+
+export const Default = Template.bind({});
```

Comment:
This might just be personal preference but when there's only one story using a template, I'd probably just skip the template altogether. Up to you though.

Reply:
I updated this- good call. I attempted doing the same thing with TabView, but it was being finnicky so I left it.

Reply:
Aaand a small update- I updated inputPrompt in a way that still looked right on the 9001 site, but the storybook unit tests are expecting a different format. For ease/quickness I switched it back to the format that last passed a build.